docs/src/use/configure/rules.md

```diff
@@ -251,6 +282,7 @@ foo(); /* eslint-disable-line example/rule-name */
 
 #### Comment descriptions
 
+In cases where a specific rule consistently causes issues that are difficult to resolve, consider creating an exemption for that rule in the project's ESLint configuration with appropriate justifications.
```

Comment:
This is a bit vague and doesn't use language that is familiar to ESLint users. What is an "exemption"? Are you saying to modify the rule options in the configuration file? If so, we should just say that. Also note that configuration files don't support reasons for disabling comments.